### DISCUSSION TITLE
Tag와 Light 문제 질문

### DISCUSSION THREAD

**Discussion by diddntjd99** [naverz/zepeto-studio-kor#78] [5. Q&A]
안녕하세요,
1. Tag 문제 - 저희 게임에서 게임종료 판단 시 Item 태그를 가진 아이템들을 모두 삭제하도록 코딩을 하였고, 실행 시 컴퓨터 내에선 삭제가 되는 것을 확인 했는데, 핸드폰에서 qr코드나 테스트 월드로 테스트 시 삭제되지 않습니다.

2. Light 문제 - 컴퓨터나 갤럭시 기종의 핸드폰으로 qr코드 실행 시 저희가 설정한대로 라이트가 제대로 적용되어 있는 것을 확인할 수 있는데, 아이폰으로 테스트 월드로 접속해보면 라이트가 제대로 적용이 되어있지 않는 것 같습니다.

3. 지난번에 캐릭터 teleport 함수 사용 문제로 질문을 남겼었는데, 문제되는 정보가 부족하다고 말씀해주셔서 그것도 추가로 질문하겠습니다. 일정 타임이 되면 캐릭터의 위치를 옮기기 위해 텔레포트 함수를 사용한건데, 여러번 플레이를 해보면 이동이 될 때도 있지만 안될때도 있습니다. 
const zepetoPlayer = ZepetoPlayers.instance.GetPlayer(this.room.SessionId);
zepetoPlayer.character.Teleport(new UnityEngine.Vector3(0, 1, -12.8), UnityEngine.Quaternion.Euler(0, 0, 0));
사용한 코드는 이렇습니다.

이 부분들 확인 부탁드리겠습니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요. 문의 주신 내용들 최신 버전에서도 발생하는지 확인 부탁 드립니다.
감사합니다.
